Common/RotationalEncoder.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cmath>
+#include <cstdint>
```

Comment:
I don't see you using cmath or cstdint in this file. Maybe you can remove these.